src/scripts/components/VariableMessage.jsx

```diff
@@ -210,6 +210,30 @@ class VariableMessage extends React.Component {
     }
   }
 
+  handleKeyUp = (k) => {
+    const backspaceKeyPressed = k.which === 8;
```

Comment:
Sorry, I meant the key code value. So in the constants file you might create a section for key codes and create a constant named `BACKSPACE_KEY`:
```
export const BACKSPACE_KEY = 8;
```

The idea is to not have any "magic numbers" in code. So when reading this code it reads:
```
if (k.which === BACKSPACE_KEY) [...]
```